platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/inputs/undeclared/FileCollectionContentsChangingTest.groovy

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.cc.impl.inputs.undeclared
+
+import org.gradle.internal.cc.impl.AbstractConfigurationCacheIntegrationTest
+
+class FileCollectionContentsChangingTest extends AbstractConfigurationCacheIntegrationTest {
```

Comment:
❓ Could you please add a test with some known location too? Now, when the file list dump happens regardless